core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -178,17 +409,30 @@ def _generate_stats(
                 content_word_count: int. The word count of the content of
                     the suggestion.
                 last_updated_date: str. When was the suggestion last updated.
+                created_date: str. When was the suggestion created.
         """
         # When opportunity is not available we leave the topic ID empty.
         topic_id = ''
         if opportunity is not None:
             topic_id = opportunity.topic_id
 
         for suggestion in suggestions:
-            key = (
-                suggestion_models
-                .TranslationContributionStatsModel.construct_id(
-                    suggestion.language_code, suggestion.author_id, topic_id))
+            if model == suggestion_models.TranslationContributionStatsModel:
+                key = model.construct_id(
+                    suggestion.language_code,
+                    suggestion.author_id,
+                    topic_id
+                )
+            else:
+                # Here we use MyPy ignore because in earlier steps of
+                # the job, we filtered the suggestions that were
+                # reviewed. Hence the final_reviewer_id should exists
+                # and not a None type.
+                key = model.construct_id(
+                    suggestion.language_code,
+                    suggestion.final_reviewer_id, # type: ignore[arg-type]
+                    topic_id
+                )
```

Comment:
I don't think this comment was addressed? https://github.com/oppia/oppia/pull/17530#discussion_r1153617288

Reply:
Done